README.md

```diff
@@ -221,5 +221,20 @@ extension SceneDelegate: TurboNavigationDelegate {
         return nil
     }
 }
+```
+
+You can also override the default WKWebViewConfiguration change using the following:
 
+
+```
+extension SceneDelegate: TurboNavigationDelegate {
```

Comment:
So something else I was thinking about. 

Now that makeWebView is here, we can also have an example of integrating a script message delegate. This would solve most of my needs anyway. 

This allows developers to have a good set of defaults and a framework to build off for the javascript brige

Reply:
An example of a script message delegate would definitely be welcome. And when Strada gets released showing how integrate there would be awesome, too.